### PR TITLE
add function to get current user

### PR DESCRIPTION
**Pull request by jaswdr**
Sometimes I want to get the current user, to write a log file or named something based on the name of the current user, to do this I've add the 'getCurrentUser()' function to the Shield class

### CODE REVIEW

**Comment by vinkla** [comment]
Thanks for your pull request. Could you describe a use case for this?

**Comment by jaswdr** [comment]
Of couse,

If I want to use the Laravel\Guard for example, knowing wich user is logged, I can block or allow some areas of my application. Example: admin\user credentials.

Another one is for logging, knowing wich user is, I can log each action and after that separate per user, so if something get wrong I know exactly wich user was on that time.

I thing the **key** of the array in **config/shield.php** is a simple and good way to nominate the user and have a reference to him

File: src/Shield.php [hunk 1]
```diff
@@ -28,6 +28,13 @@ class Shield
     protected $users;
 
     /**
+     * The current user authenticated.
```
Comment (vinkla) [inline src/Shield.php]: Please change to `The authenticated user.`

File: tests/ShieldTest.php [hunk 2]
```diff
@@ -34,6 +34,17 @@ public function testVerifyWithUser()
         $this->assertNull($shield->verify('user1', 'password1', 'main'));
     }
 
+    public function testGetUserAfterVerify()
```
Comment (vinkla) [inline tests/ShieldTest.php]: Please change to `testGetCurrentUser()`.

**Comment by jaswdr** [comment]
@vinkla everything updated :+1:

**Comment by vinkla** [comment]
@jaschweder Good work :+1: Now please [squash your commits](https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/) and then I'll merge this in.

**Comment by jaswdr** [comment]
finished

**Comment by vinkla** [comment]
Thanks!

**Comment by vinkla** [comment]
Released it as 0.3.2

**Pull request merged**